lms/djangoapps/grades/models.py

```diff
@@ -121,6 +123,7 @@ def create_from_blockrecords(self, blocks):
         """
         model, _ = self.get_or_create(
             hashed=blocks.hash_value,
+            version=VISIBLE_BLOCKS_VERSION,
```

Comment:
I believe this won't do an indexed query since version is not part of the index.